build.sh

```diff
@@ -508,12 +538,22 @@ case "$COMMAND" in
         ;;
 
     "test-ios")
-        xctest Realm -configuration "$CONFIGURATION" -sdk iphonesimulator -destination 'name=iPhone 11'
+        if [ "$CI" ]; then
+            change_schema_configuration Realm
+        else
+            xctest Realm -configuration "$CONFIGURATION" -sdk iphonesimulator -destination 'name=iPhone 11'
+        fi
```

Comment:
I don't think it makes sense to try to combine the xcode cloud and manual run targets like this, since it's just doing two completely unrelated things depending on the state of an env variable.

Updating the scheme's build configuration could probably just be done in ci_post_clone.sh? You could loop over all of the schemes and set them to the appropriate configuration regardless of which target is being built.

"schema" is being used here where the correct word is "scheme".